cmd/maya-agent/types/v1/block_device_info.go

```diff
@@ -0,0 +1,37 @@
+package app
+
+//"k8s.io/apimachinery/pkg/runtime/serializer/json"
+
+//OsCommand is for operating system related commands for block devices
+type OsCommand struct {
+	Command string
+	Flag    string
+}
+
+//BlockDeviceInfo exposes the json output of lsblk:"blockdevices"
+type BlockDeviceInfo struct {
+	Blockdevices []Blockdevices `json:"blockdevices"`
+}
+
+//Blockdevices has parent block disk details
+type Blockdevices struct {
+	Name       string     `json:"name"`
+	Majmin     string     `json:"maj:min"`
+	Rm         string     `json:"rm"`
+	Size       string     `json:"size"`
+	Ro         string     `json:"ro"`
+	Type       string     `json:"type"`
+	Mountpoint string     `json:"mountpoint"`
+	Children   []Children `json:"children,omitempty"`
```

Comment:
should we have it as children or have a parent tag -- refering to BlockDevice itself?